### DISCUSSION TITLE
Welcome to add your FixIt site | 欢迎添加你的 FixIt 网站

### DISCUSSION THREAD

**Discussion by Lruihao** [hugo-fixit/FixIt#120] [Show and tell]
Welcome to add your FixIt site to [this page](https://fixit.lruihao.cn/friends/) via [making a PR ](https://github.com/hugo-fixit/docs/pulls).

 > ⚠️ Website failure, stop maintenance and improper content may be unlinked!

---

您可以通过 [创建 PR ](https://github.com/hugo-fixit/docs/pulls) 将您的 FixIt 网站添加到[此页面](https://fixit.lruihao.cn/zh-cn/friends/)。

 > ⚠️ 网站失效、停止维护、不当内容都可能被取消链接！

**Comment by Lruihao** [comment]
comment site info at PR body like `friends.yml` format:

````markdown
```yaml
- nickname: <your nickname>
   avatar: <your avatar>
   url: <your site link>
   description: <description of your site>
```
````

**Comment by acuherb** [comment]
- nickname: 针草堂
  avatar: https://acuherb.xyz/images/avatar.jpg
  url: https://acuherb.xyz
  description: 针通经络，草养气血。

**Comment by bulone** [comment]
nickname: 吐司气泡
avatar: https://blog.toastbubble.top/logo.svg
url: https://blog.toastbubble.top
description: 🍞这有多的吐司速来品鉴

**Reply by bulone** [reply]
申请修改头像链接为https://blog.toastbubble.top/avatar.webp

**Reply by Lruihao** [reply]
已调整

**Comment by ah0210** [comment]
nickname: 自游人
avatar: https://17you.com/icons/web-app-manifest-512x512.png
url: https://17you.com/
description: 旅行和写作

**Comment by Monkeyke** [comment]
nickname: MonkeyKe’s Blog
avatar: https://cravatar.cn/avatar/dee014ecea51d0dc08dbb1aaec046b47
url: https://monkeyke.com
description: 一个热爱探索的开发者

**Reply by Monkeyke** [reply]
更新一下头像链接：https://cravatar.cn/avatar/dee014ecea51d0dc08dbb1aaec046b47?s=512

**Comment by pullsec** [comment]
- nickname: PullSec
  avatar: https://pullsec.io/images/cat.jpg
  url: https://pullsec.io
  description: Security research, systems engineering, vulnerability analysis, and hands-on experimentation.
